### ISSUE TITLE
HW wizard should disable triggers for low gain PMTs and channels with no PMT attached

### ISSUE THREAD

**Issue by tlatorre-uchicago** [snoplus/orca#278]
This is really a database issue, but putting it here as a reminder. The database column which specifies whether a PMT is at HV should include channels with no PMT and low gain PMTs.

**Comment by tlatorre-uchicago** [comment]
@fbdescamps how easy is it to add this check to your conversion script?

**Comment by tlatorre-uchicago** [comment]
Just realized this needs to be separate from the pulled resistor and pulled cable column since we might want to treat them differently.

**Comment by fbdescamps** [comment]
Yes, there is a column now for the low gain tubes in the pmtdb table in detector db. This is currently all zeros but should be updated soon. I'll post a comment here when it is done.

**Comment by tlatorre-uchicago** [comment]
This can be solved pretty easily now, but I need to talk with Phil about how we want to implement the HW wizard logic. Should it just use the nominal settings and not enable triggers for channels whose nominal setting is off?
